### DISCUSSION TITLE
Poetry and SLSA

### DISCUSSION THREAD

**Discussion by fretchen** [python-poetry/poetry#8958] [Q&A]
It would seem that [SLSA](https://github.com/slsa-framework/slsa-github-generator) is a framework to make it safer to work with open source packages. Now it would also seem to have an integration with releases on github. However, the release procedure of poetry is quite nice and I started to wonder if someone has already had a look if the two could come together...

**Comment by abn** [comment]
I don't think there is any current work on this yet. I suspect it is a good thing to make available.

There is some existing work around enabling this for python packages at https://sethmlarson.dev/python-and-slsa. If you are interested, feel free to create a new issue with the proposal and have a crack at a solution perhaps.

Happy to help it along.

**Reply by fretchen** [reply]
Thanks for the feedback. I [opened the issue](https://github.com/python-poetry/poetry/issues/9069) as suggested. However, given that I barely got beyond using poetry until now I have doubts about my capabilities to integrate it :)
